tests/unit/disco/components/TestAddon.js

```diff
@@ -319,8 +359,9 @@ describe('<Addon />', () => {
       themeImage = findDOMNode(root).querySelector('.theme-image');
       const preventDefault = sinon.spy();
       Simulate.click(themeImage, { preventDefault });
+
       expect(preventDefault.called).toBeTruthy();
-      expect(installTheme.calledWith(themeImage, data.addon)).toBeTruthy();
+      sinon.assert.calledWith(installTheme, themeImage);
```

Comment:
This was actually a case of "how did this test ever work?" since updating the `withInstallButton` HOC. I fixed it up using enzyme... but we should definitely clean up this test suite in general...